src/_helpers.ts

```diff
@@ -0,0 +1,27 @@
+import Toastify from 'toastify-js';
+
+const showToast = (variant: 'success' | 'warning' | 'error', text: string) => {
+	const style = {
+		success: 'linear-gradient(to right, #00b09b, #96c93d)',
+		warning: '#fd9900',
+		error: '#fd3a00',
+	};
+
+	const background = style[variant];
+
+	Toastify({
```

Comment:
Ogólnie jest super. Jezeli masz jeszcze chwile czasu, dla samego treningu TSa, mozesz dodac do funkcji showToast parametr `options` który będzie umozliwial nadpisanie wszystkich opcji toastify na zyczenie. 
Wtedy musisz otypować options jako obiekt tego samego typu który przyjmuje funkcja Toastify, jedynie opakować options za pomoca Partial<ToastifyConfig> (przykladowo). Wtedy funkcja przyjmie parametry znane z configu toastify i bedziesz mogl je nadpisywać (na koncu obiektu konfiguracyjnego dodasz tylko {...options} i tyle. 

Takie cos pozwala na ominiecie powielania wszystkich opcji z toastify w funkcji ktora tworzysz, a bedziesz mogl przekazywac wszystko to, co chcesz nadpisać.